docs/DEVELOPMENT.md

```diff
@@ -279,7 +279,7 @@ See [test/e2e/README.md](/test/e2e/README.md)
 You can change log level controllers for development purpose. Edit configmap:
 
 ```sh
-kubectl edit cm config-logging -n tekton-pipelines
```

Comment:
Miss `tekton-results-` here.  `config/components/metadata/kustomization.yaml` set this prefix